reviewer_readme.md

```diff
@@ -1,46 +1,116 @@
+###### Preview the submission
 
-### Automated Testing
+1. Click "open this in GitHub Desktop" on the pull request page.
+2. Open a Terminal Window by using the menu: Repository > Open in Terminal.
+3. type in and run the following command: `roots watch`. This will build the site and open the site in a web browser.
+4. Find and Preview the new article and double check author's details.
 
-Travis CI will automatically trigger jobs that verify that the author's content is sane enough and that the result of the merge will build into a "valid" site. We recommend that this only be used as a guide line and not as a definitive solution. Automated test suites can always be better, and reviewers should look at the results of http://int-androidhub.intel.com before pushing that to production.
+![open in github](https://drive.google.com/uc?id=0BwL_X3dS1kN_a2FUYUFKdHNZSGc)
 
-### Denying Content
 
-To deny a pull request, click the "Close Pull Request" Button. The author will be notified via email, but we encourage you to also send a message as to why their contribution is not going to be merged into the site.
+--------
 
-### Merging Content
+### Step 3: Review tests
 
-Discuss the content with the author in the pull request. When you both agree that the content is ready, and Travis CI jobs are green, it is safe to merge.
+![failed tests](https://drive.google.com/uc?id=0BwL_X3dS1kN_NmpXbDFZMGdIOTg)
 
-### View on Integration site
+The author's submission will automatically be tested by a product called Travis CI that will verify if the submission is valid. Submissions that fail these tests should be rejected (aka. closed). Just because a submission passes these tests does not mean you absolutely should accept (merge) it. Click on "Details" for more information. We recommend that automated tests be used only as a guideline and not as a definitive solution. Automated test suites can always be better, and reviewers should look at the results of http://int-androidhub.intel.com before releasing to production.
+
+--------
+
+### Step 4: Accept or Deny
+
+##### Accept
+
+![ready to merge](https://drive.google.com/uc?id=0BwL_X3dS1kN_eFVlVktXVXlhUG8)
+
+If a submission's content looks good and the Travis CI tests are green, it is safe to merge. Click the button on the Pull Request page labeled "Merge pull request". This will merge the content into the `master` branch and tell Travis CI to automatically deploy the site to the private Integration environment.
+
+##### Deny
+![pr comment](https://drive.google.com/uc?id=0BwL_X3dS1kN_cGJiaWRwb05Dbzg)
+
+To deny a submission, click the "Close Pull Request" Button. The author will be notified via email. We encourage you to also send a message as to why the submission is not going to be merged into the site at this time.
+
+--------
+
+### Step 5: Auto-Deploy to Integration
```

Comment:
Intel named the url "int-androidhub.intel.com" and "int" is meant to be integration, so I expect its a term they use. Normally I'd use staging as well.